lib/utils.js

```diff
@@ -7,98 +10,79 @@ var assert = require("assert");
  * @returns Configuration Object
  */
 exports.parseMongoConnectionURL = function (mongoConnectionURL) {
-  var result = {};
 
-  //The connection string will be of the form mongodb://user:password@host:port,host2port2/databasename?option=someOption
+  //The connection string will be of the form mongodb://username:pass@db1.prv.mbaas2.cluster.feedhenry.net,db2.prv.mbaas2.cluster.feedhenry.net:27017/domain-vvuq7douloe2zkyktub7fcmt-demos-dev?replicaSet=cluster-mbaas2_rs1
```

Comment:
Before it was with the description of how the connection string is formed. You made this changes and shows put real values of the env FH_MONGODB_CONN_URL which is just valid for upgraded databases. 
I think it should be into the previous format instead of an example with real values.